en/guide/camera_definition_file.md

```diff
@@ -2,15 +2,16 @@
 
 As defined in the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html), a [Camera Definition File](https://mavlink.io/en/protocol/camera_def.html) may be used to specify information about a camera's advanced settings and parameters. A camera will supply the URI for its definition file on request (if one has been defined), so that the file can be downloaded and parsed by the client.
 
+> **Tip** Camera definition files for a number of cameras are provided in [/samples/def](https://github.com/intel/camera-streaming-daemon/tree/master/samples/def). Definition files for attached Video4Linux (V4L2) devices can be create using [export_v4l2_param_xml](#export_v4l2_param_xml).  
+
 The *Camera Definition Files* can be hosted anywhere that is accessible to clients (including *QGroundControl* and Dronecode SDK apps). Often the files are served from the computer running CSD.
 
 CSD is able to supply the URIs for attached cameras using a mapping given in the [CSD Configuration File > \[uri section\]](../guide/configuration_file.md#uri).
 
 This topic explains how to:
 * Serve the definition files on the CSD companion computer.
 * Map attached cameras to their definition file URIs in the CSD configuration file.
-
-> **Tip** Camera definition files for a number of cameras are provided in [/samples/def](https://github.com/intel/camera-streaming-daemon/tree/master/samples/def).
+* Create *Camera Definition Files* for attached Video4Linux (V4L2) devices using the *export_v4l2_param_xml* tool. 
```

Comment:
Please check if the sentence proposed below sound better
> Create *Camera Definition Files* for attached Video4Linux (V4L2) devices using the *export_v4l2_param_xml* tool

Create *Camera Definition Files* for Video4Linux (V4L2) camera devices attached to linux machine using the *export_v4l2_param_xml* tool

Reply:
Well it doesn't sound better, but it is more accurate. I'll update :-)

Reply:
The fact that it is Linux is implied by the Video4Linux API, and we say "attached". That said, I have improved this a bit:

> Additionally, the [export_v4l2_param_xml](#export_v4l2_param_xml) tool can automatically generate definition files for attached Video4Linux (V4L2) camera devices.